[2025-02-16T08:19:25.829+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: pipeline_group1.extract_data manual__2025-02-16T08:19:22.764464+00:00 [queued]>
[2025-02-16T08:19:25.848+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: pipeline_group1.extract_data manual__2025-02-16T08:19:22.764464+00:00 [queued]>
[2025-02-16T08:19:25.850+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2025-02-16T08:19:25.850+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 1
[2025-02-16T08:19:25.851+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2025-02-16T08:19:25.878+0000] {taskinstance.py:1389} INFO - Executing <Task(PythonOperator): extract_data> on 2025-02-16 08:19:22.764464+00:00
[2025-02-16T08:19:25.887+0000] {standard_task_runner.py:52} INFO - Started process 454 to run task
[2025-02-16T08:19:25.889+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'pipeline_group1', 'extract_data', 'manual__2025-02-16T08:19:22.764464+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/P2M3_inka_DAG.py', '--cfg-path', '/tmp/tmpxoj_prmp', '--error-file', '/tmp/tmptwn8iq8i']
[2025-02-16T08:19:25.890+0000] {standard_task_runner.py:80} INFO - Job 6: Subtask extract_data
[2025-02-16T08:19:25.962+0000] {task_command.py:371} INFO - Running <TaskInstance: pipeline_group1.extract_data manual__2025-02-16T08:19:22.764464+00:00 [running]> on host 8d7c06d84acb
[2025-02-16T08:19:26.044+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=group1
AIRFLOW_CTX_DAG_ID=pipeline_group1
AIRFLOW_CTX_TASK_ID=extract_data
AIRFLOW_CTX_EXECUTION_DATE=2025-02-16T08:19:22.764464+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2025-02-16T08:19:22.764464+00:00
[2025-02-16T08:19:26.061+0000] {base.py:68} INFO - Using connection ID 'postgres_***' for task execution.
[2025-02-16T08:20:23.842+0000] {local_task_job.py:156} INFO - Task exited with return code Negsignal.SIGKILL
[2025-02-16T08:20:25.139+0000] {taskinstance.py:1412} INFO - Marking task as FAILED. dag_id=pipeline_group1, task_id=extract_data, execution_date=20250216T081922, start_date=20250216T081925, end_date=20250216T082025
[2025-02-16T08:20:25.672+0000] {local_task_job.py:279} INFO - 0 downstream tasks scheduled from follow-on schedule check
